### DISCUSSION TITLE
Pretty cool case

### DISCUSSION THREAD

**Discussion by adocampo** [OpenVoiceOS/ovos-buildroot#97] [Show and tell]
Just bought a pretty cool case for rpi on Aliexpress. 
It just misss a camera holder to be really cool, and speakers, well, could be more powerful, but for someone who doesn't have  a 3D printer like me, and wants to have everything on the same case, it can be interesting.
I did a quick video for you to take a look.
[![Open in youtube](https://img.youtube.com/vi/H3SocLOJegk/0.jpg)](https://www.youtube.com/watch?v=H3SocLOJegk)

**Comment by JarbasAl** [comment]
thanks for sharing! For sure the best custom OVOS build i have seen so far, everyone else is just using the mark2 dev kit :)

**Comment by adocampo** [comment]
Details of the physical buttons
![1635115282126](https://user-images.githubusercontent.com/2266612/138615922-b8840fb2-924d-4b6c-9d39-538ca84ed22c.jpg)

Rpi4 side: with usb micro and touch screen connected (there are also the possibility of soldering the touch screen connector on the rpi motherboard (instructions for rpi3/4 and cable provided)
![1635115282129](https://user-images.githubusercontent.com/2266612/138615964-7fc27d82-41f3-4ed2-b65f-eb553669ce9a.jpg)

Case motherboard connections, it replaces the type-c rpi4 power plug with a 12v adaptor, a type-c usb, hdmi and audio jack.
![1635115573724](https://user-images.githubusercontent.com/2266612/138615989-e177c61e-0c51-47b0-9da5-70ab33a88101.jpg)

I finally discarded the camera, couldn't find a proper way to make it available outside the case, and I need to work with the fan, as it is quite noisy (just bought some dupont 1-pin female connectors and will plug the fan directly to the rpi 3vcc output, instead of the 12v where is currently connected)

**Comment by AIIX** [comment]
oh wow, thanks a ton for sharing! I was looking at the same case + screen from Aliexpress for building a DIY OVOS setup and it seems to be working really nicely, I thought this one does not come with a microphone inbuilt or are you using some external mic array? I was wondering if the empty slot next to the fan can possibly hold a small mic array

**Reply by adocampo** [reply]
I'm using a very very very cheap usb micro dongle (around 1,5€), but it works pretty decent if there are no sound playing it OVOS. 
Surely you can allocate a mic array on inside the case (there are no screw holes on the other slot but with I think there are enough room for a mic array), I didn't try because of the respeaker issue. The main problem is the internal fan, which is quite noisy, my bet is because is running at 12v. On another pi4 I had to use the 3vcc output to make its fan silent, I guess we should do something like that in here, but I don't know if taking those two jumpers we would be able to put a hat in there. Or perhaps you can simply unplug the fan as I doubt it will make any change at all.
Said that, I do not know the exact dimension of the different speaker arrays and/or if there is another way to connect to the mobo different from a rpi hat.

Apart from that, screen is awesome, it has a good resolution, it responds to touch quick and precisely and there are no dead pixels or whatsoever.  Sound could be better, indeed, I need to find some time to tear down the case and disassemble the speakers, I see there are room for bigger ones (at least, a bit bigger ones)

**Comment by AIIX** [comment]
The physical buttons seem to have their own software control which seems related somehow to the display board, maybe they can be wired to the software controls on the pi via gpio

**Comment by ChanceNCounter** [comment]
I thought I commented on this last night from my phone, but I guess it didn't post. This is fabulous! I think, though I could be wrong, that you're the first person not named j1nx or Daniel to build a box.
